modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -1,36 +1,34 @@
 // Module included in the following assemblies:
 //
 // * applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc
+
 :_content-type: PROCEDURE
 [id="odc-creating-a-binding-connection-between-components_{context}"]
 = Creating a binding connection between components
 
-You can establish a binding connection with Operator-backed components.
-
-This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
-The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator. The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
```

Comment:
I'm sorry, I meant to raise this point earlier, but I have a few issues with the first sentence here:

`This procedure` is not in line with page 54 of **ibm_style_Q2_2022**:
_"Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes"."_

Also, `This procedure walks` is an anthropomorphism, which is not in line with page 77 of **ibm_style_Q2_2022**:
_"Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions."_
The procedure doesn't have legs and cannot walk. It's pretty clear to me that intended meaning was to `walk you through an example`, but `you` is missing here, and even so I don't recommend using this phrase in the docs because it's not the literal meaning and as such is not good for international audiences (this isn't a text about walking).

Reply:
BTW, I previously suggested a solution to this, but my suggestion was discarded:
https://github.com/openshift/openshift-docs/pull/49201#discussion_r962729192

Reply:
@Dhruv-Soni11 Please replace the 1st sentence "This procedure ...application." with this: "You can create a binding connection with Operator-backed components, as demonstrated in the following example, which uses a PostgreSQL Database service and a Spring PetClinic sample application in the YAML view.

Reply:
@Srivaralakshmi - Thanks for suggesting this change. I have gone ahead and made it in the content.